pkg/tuned/cmd/render/cmd.go

```diff
@@ -57,6 +58,7 @@ func NewRenderBootCmdMCCommand() *cobra.Command {
 func (r *renderOpts) AddFlags(fs *pflag.FlagSet) {
 	fs.StringArrayVar(&r.assetsInDir, "asset-input-dir", []string{components.AssetsDir}, "Input path for the assets directory. (Can use it more than one to define multiple directories)")
 	fs.StringVar(&r.assetsOutDir, "asset-output-dir", r.assetsOutDir, "Output path for the rendered manifests.")
+	fs.StringVar(&r.mcpName, "mcp-name", r.mcpName, "MachineConfigPool name for current node.")
```

Comment:
```suggestion
	fs.StringVar(&r.mcpName, "mcp-name", r.mcpName, "MachineConfigPool name for the current node.")
```

We could maybe (just a q, no change requested yet :) ask for a node role name instead and detect the MCP. But the MCP name is a good start.

Reply:
Yeah sure ... whatever could be considered easier from user perspective.